test/form/samples/system-multiple-export-bindings/_expected.js

```diff
@@ -0,0 +1,37 @@
+System.register([], function (exports) {
+	'use strict';
+	return {
+		execute: function () {
+
+			const x = exports('x', 123);
+			const y = exports('y', 456);
+
+			var namespace = /*#__PURE__*/Object.freeze({
+				__proto__: null,
+				x: x,
+				y: y
+			});
+			exports({namespace: namespace, namespace2: namespace});
+
+			// Namespace variable
+
+			// Variable Declaration
+			let a = exports('default', exports('a2', exports('a', 1))), b = exports('b', 2), c = exports('c2', exports('c' = 3));
```

Comment:
@guybedford and @lukastaegert, could you guys validate the expectations in this test case for correctness? Are there any test cases missed here that are relevant?

Reply:
This looks good.